suite-common/wallet-utils/src/formDraftUtils.ts

```diff
@@ -5,9 +5,12 @@ export const getFormDraftKey = (prefix: FormDraftKeyPrefix, key: string) => `${p
 export const parseFormDraftKey = (
     formDraftKey: string,
 ): [prefix: FormDraftKeyPrefix, key: string] => {
-    const strings = formDraftKey.split('/');
-    if (strings.length === 2) {
-        return strings as [prefix: FormDraftKeyPrefix, key: string];
+    const delimiter = formDraftKey.indexOf('/');
```

Comment:
Just a nitpick... I'd go with `delimiterIndex` or similar since it's not a delimiter in fact.

Reply:
https://github.com/trezor/trezor-suite/pull/12929/commits/223eefde583e92fc248361df94fc5d665613cf8a